src/SFA.DAS.ApprenticeCommitments/Map/RegistrationModelExtensions.cs

```diff
@@ -12,8 +12,10 @@ public static Registration MapToRegistration(this RegistrationModel registration
                 Id = registrationModel.Id,
                 ApprenticeshipId = registrationModel.ApprenticeshipId,
                 Email = registrationModel.Email,
+                EmployerName = registrationModel.EmployerName,
+                AccountLegalEntityId = registrationModel.AccountLegalEntityId,
                 UserIdentityId = registrationModel.UserIdentityId,
-                ApprenticeId = registrationModel.Id,
+                ApprenticeId = registrationModel.Id
```

Comment:
My personal style is to always include the trailing commas - it means that future source-control diffs will not flag these lines as changed when adding new properties - because the don't have to change :)